maven/README.md

```diff
@@ -32,6 +32,10 @@ version of the module.
 * Any changes that have been made to the packages exported by the bundle.
 * Any internal changes within the bundle.
 
+## [bnd-export-maven-plugin][4]
```

Comment:
Ref [4] points to the travis.yml file. You need to add a ref for the readme for this plugin.